src/components/metadataEditor/metadataEditor.js

```diff
@@ -38,35 +38,32 @@ function closeDialog() {
     }
 }
 
-function submitUpdatedItem(form, item) {
-    function afterContentTypeUpdated() {
+async function submitUpdatedItem(form, item) {
+    const afterContentTypeUpdated = () => {
         toast(globalize.translate('MessageItemSaved'));
 
         loading.hide();
         closeDialog();
-    }
-
+    };
```

Comment:
This is only called once now, so please remove the function and inline the contents.